### ISSUE TITLE
Email Notifications Broken - ERROR NotificationService

### ISSUE THREAD

**Issue by ctgaff** [rundeck/rundeck#1181]
Using Oracle Java 7 with Release 2.5.0-1.

All attempts within jobs to automatically send email notifications generate the error below. I also tried a new install with a new Job and obtained the same error.  I am using the same format as 2.4.x where "rundeck-config.properties" contains `grails.mail.host=<mailhost>`.

```
ERROR NotificationService - Error sending notification email to <email address> for Execution 1: Error executing tag <g:render>: Error executing tag <g:render>: Error executing tag <g:render>: Error executing tag <g:render>: Error executing tag <g:render>: Error executing tag <stepplugin:display>: You cannot use the session in non-request rendering operations
```

Thanks,
Corey

**Comment by gentunian** [comment]
I'm wondering if Im suffering this too...

I have a QA rundeck environment. I moved that to production. Changed a few things in config files and I've got that error:

```
2015-06-23 16:49:46,222 ERROR NotificationService - Error sending notification email to
<maildir> for Execution 4: Error executing tag <g:render>: Error executing tag <g:render>: 
Error executing tag <g:render>: Error executing tag <g:render>: Error executing tag
<g:render>: Error executing tag <stepplugin:display>: You cannot use the session 
in non-request rendering operations
```

In QA mails are working OK. This are the java versions of each:

QA:
```
    java -version
    java version "1.7.0_75"
    OpenJDK Runtime Environment (rhel-2.5.4.2.el7_0-x86_64 u75-b13)
    OpenJDK 64-Bit Server VM (build 24.75-b04, mixed mode)
```
PRODUCTION:
```
    java -version
    java version "1.7.0_45"
    Java(TM) SE Runtime Environment (build 1.7.0_45-b18)
    Java HotSpot(TM) Client VM (build 24.45-b08, mixed mode, sharing)
```

They are slightly different JREs.

Rundeck version (2.5.0) is the same as I've `scp`d to the production host.

**Comment by gschueler** [comment]
@gentunian this was supposed to be fixed in 2.5.1

**Comment by gentunian** [comment]
My bad then, I missed the part of "ORACLE JDK". Seems that in OpenJDK 1.7 problem is not showing.
